engine/src/flutter/lib/web_ui/lib/src/engine/platform_dispatcher/view_focus_binding.dart

```diff
@@ -5,20 +5,13 @@
 import 'dart:async';
 import 'dart:js_interop';
 
-import 'package:meta/meta.dart';
 import 'package:ui/src/engine.dart';
 import 'package:ui/ui.dart' as ui;
 
 /// Tracks the [FlutterView]s focus changes.
 final class ViewFocusBinding {
   ViewFocusBinding(this._viewManager, this._onViewFocusChange);
 
-  /// Whether [FlutterView] focus changes will be reported and performed.
-  ///
-  /// DO NOT rely on this bit as it will go away soon. You're warned :)!
-  @visibleForTesting
-  static bool isEnabled = true;
```

Comment:
I remember @tugorez having either an open PR for this, or an issue that should be marked as "fixed"/closed by this PR.